Module4_Labs/Lab1_BFS_Maze/Cards/5.md

```diff
@@ -0,0 +1,60 @@
+# Time and Space Analysis of Breath First Search
+
+In this section we are going to perform an analysis of the runtime and space complexity of the BFS search. 
+
+## Time complexity of BFS
```

Comment:
This info should be in the BFS activity and then the lab checkpoint should just test their knowledge.